kwok/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM registry.k8s.io/kwok/cluster:v0.3.0-k8s.v1.27.3
+
+COPY user-signups.sh ./user-signups.sh 
+
+RUN chmod +x ./user-signups.sh
+
+COPY entrypoint.sh ./entrypoint.sh
+
+RUN wget https://raw.githubusercontent.com/codeready-toolchain/host-operator/master/config/crd/bases/toolchain.dev.openshift.com_usersignups.yaml
```

Comment:
please don't curl/wget doing image build - either copy that file to our repo or link it via git submodules.

Reply:
Done